test-docker-compose.yaml

```
@@ -63,3 +63,19 @@ services:
       - '.env'
     environment:
       - 'KEEP_ALIVE_PERIOD=0'
+
+  unit_test:
+    container_name: 'kernelci-api-unittests'
+    build:
+      context: 'docker/api'
+      args:
+        - REQUIREMENTS=${REQUIREMENTS:-requirements-dev.txt}
+    volumes:
+      - './api:/home/kernelci/api'
+      - './tests:/home/kernelci/tests'
```

This should only need `unit_tests` directory.
```suggestion
      - './tests:/home/kernelci/tests/unit_tests'
```